src/test/BurnableMintableCappedERC20Init.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { IAxelarGateway } from '../interfaces/IAxelarGateway.sol';
+import { IBurnableMintableCappedERC20 } from '../interfaces/IBurnableMintableCappedERC20.sol';
+
+import { MintableCappedERC20 } from '../MintableCappedERC20.sol';
+import { DepositHandler } from '../DepositHandler.sol';
+
+contract BurnableMintableCappedERC20Init is IBurnableMintableCappedERC20, MintableCappedERC20 {
```

Comment:
Would it be possible just to extend `BurnableMintableCappedERC20` and define only constructor and init?

Reply:
Done

Reply:
done.